docs/app/Components/Sidebar/Sidebar.js

```diff
@@ -11,23 +11,23 @@ export default class Sidebar extends Component {
   handleSearchChange = e => this.setState({ query: e.target.value })
 
   getComponentsByQuery() {
-    return _.filter(stardust, component => {
+    return _.filter(component => {
       const name = component._meta.name
       const isParent = META.isParent(component)
       const isQueryMatch = new RegExp(this.state.query, 'i').test(name)
       return isParent && isQueryMatch
-    })
+    }, stardust)
   }
 
   getComponentsByType = type => {
-    const items = _(this.getComponentsByQuery())
-      .filter(component => META.isType(component, type))
-      .sortBy((component, name) => name)
-      .map(component => {
+    const items = _.flow(
+      _.filter(component => META.isType(component, type)),
+      _.sortBy((component, name) => name),
+      _.map(component => {
```

Comment:
Chaining is not supported, this was the one place we used it.  Refactored to use [`lodash/fp`](https://github.com/lodash/lodash/wiki/FP-Guide) now with [`_.flow`](https://lodash.com/docs#flow).

This is simply composing functions.